ts/features/mvl/screens/details/components/attachment/MvlAttachments.tsx

```diff
@@ -93,7 +93,7 @@ const MvlAttachmentItem = (props: { attachment: MvlAttachment }) => {
               color={"bluegrey"}
               weight={"SemiBold"}
               ellipsizeMode={"middle"}
-              numberOfLines={1}
+              numberOfLines={2}
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏

With this change, the component doesn't render the size (when available):

**now**:
<img width="387" alt="Schermata 2022-01-26 alle 11 18 37" src="https://user-images.githubusercontent.com/26501317/151145501-eff1ca2b-eef9-43f1-9208-a5983db770e8.png">

**before**:
<img width="384" alt="Schermata 2022-01-26 alle 11 18 58" src="https://user-images.githubusercontent.com/26501317/151145551-1b9de267-a424-4829-bb62-d0540ab34556.png">

In order to have items with a fixed height, could be a solution expand the `displayName` on two line only if the size of the attachment is not available? Otherwise we could expand the height of the button to support three lines of text.

Reply:
Good point. I fixed it by setting the item's height to 80. That's how the problem is solved on the [Figma prototype](https://www.figma.com/file/6ltZYpO1DA2EJjDoVpnsrg/UX-IO-PEC%3C%3EPN%3C%3EIO%3C%3EDomicilio-Digitale-%5Bdraft%5D?node-id=2233%3A11623) as well.

![Screenshot 2022-02-01 at 12 02 52](https://user-images.githubusercontent.com/2094604/151968657-cb48c484-3f5d-4832-8002-5571839d94f1.png)